libs/cloud/odc/aws/queue.py

```diff
@@ -180,3 +181,30 @@ def get_messages(
         raise Exception(f"Limit {limit} is not valid.")
 
     return itertools.islice(messages, limit)
+
+
+def publish_to_topic(topic_name: str, action: str, stac: dict):
+    """
+    Publish 'added' or 'archived' action to the provided sns topic
+    """
+    sns = boto3.client("sns")
+    sns_topic = sns.create_topic(Name=topic_name)
```

Comment:
It's not really the job of these tools to create infrastructure like SNS topics - we set all that up elsewhere. I think this code should accept an already existing topic (i.e., the ARN), and not create it.

Reply:
`create_topic` only creates a topic if it doesn't exist, otherwise it simply returns the arn. This is the only way to get a topic arn based off the name, though I can change the flag to take in the arn instead if that's preferable.

Reply:
Oh I see. I don't think `odc-tools` should create infra. Rather do that explicitly elsewhere. I think ARN is fine

Reply:
Done